### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by pbrianmackey** [node-ffi/node-ffi#191]
I am running windows 7.  I ran "npm install ffi --save" and got the following:

```
-

gyp ERR! configure error
gyp ERR! stack Error: Command failed:
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:647:15)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at maybeClose (child_process.js:755:16)
gyp ERR! stack     at Socket.<anonymous> (child_process.js:968:11)
gyp ERR! stack     at Socket.emit (events.js:95:17)
gyp ERR! stack     at Pipe.close (net.js:465:12)
gyp ERR! System Windows_NT 6.1.7601
gyp ERR! command "node" "D:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\bmackey\github\vim-mode\node_modules\ffi\node_modules\ref-struct\node_modules\ref
gyp ERR! node -v v0.10.29
gyp ERR! node-gyp -v v0.13.1
gyp ERR! not ok
-

gyp ERR! configure error
gyp ERR! stack Error: Command failed:
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:647:15)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at maybeClose (child_process.js:755:16)
gyp ERR! stack     at Socket.<anonymous> (child_process.js:968:11)
gyp ERR! stack     at Socket.emit (events.js:95:17)
gyp ERR! stack     at Pipe.close (net.js:465:12)
gyp ERR! System Windows_NT 6.1.7601
gyp ERR! command "node" "D:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\bmackey\github\vim-mode\node_modules\ffi\node_modules\ref
gyp ERR! node -v v0.10.29
gyp ERR! node-gyp -v v0.13.1
gyp ERR! not ok
npm ERR! ref@1.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ref@1.0.0 install script.
npm ERR! This is most likely a problem with the ref package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls ref
npm ERR! There is likely additional logging output above.

npm ERR! System Windows_NT 6.1.7601
npm ERR! command "D:\\Program Files\\nodejs\\\\node.exe" "D:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
"ffi" "--save"
npm ERR! cwd C:\Users\bmackey\github\vim-mode
npm ERR! node -v v0.10.29
npm ERR! npm -v 1.4.14
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     C:\Users\bmackey\github\vim-mode\npm-debug.log
npm ERR! not ok code 0
```

**Comment by gogins** [comment]
I believe this is related to the just prior issue.

**Comment by ot-brian-mackey** [comment]
That question is specifically in regards to a problem and fix for node v0.12.  From the output of the logs above I'm running:

```
gyp ERR! node -v v0.10.29
gyp ERR! node-gyp -v v0.13.1
```
Unless I am missing something I believe this is a separate issue.

**Comment by TooTallNate** [comment]
Do you have Python and MSVS and the necessary build tools installed?
